src/omotes_sdk/internal/worker/params_dict.py

```diff
@@ -23,7 +23,7 @@ class MissingFieldTypeException(Exception):
     ...
 
 
-PBStructCompatibleTypes = Union[List["PBStructCompatibleTypes"], float, str, bool]
+PBStructCompatibleTypes = Union[list, float, str, bool]
```

Comment:
Is there a specific reason for defining `PBStructCompatibleTypes` here instead of within `omotes_sdk/types.py`?

Reply:
None whatsoever except that I didn't move it. Good idea!

Reply:
Fixed in https://github.com/Project-OMOTES/omotes-sdk-python/pull/60/commits/51d0d3e15b0b08040dee631361ce1b7abe3119ea